package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "emails",
+  "description": "A history of communication with teams.",
+  "scripts": {
+    "test": "mdspell --en-gb -ranx 'SB2018/*.md'"
```

Comment:
Can we generalise the path?
I'm concerned that this would silently ignore issues in emails outside the SB2018 folder (for example: we don't really want to have to modify this if we create an SB2019 folder).

Reply:
I couldn't find a nice way of doing this. Simply using `**/*.md` also looks inside `node_modules`, which causes a lot of errors! (~1500)

Reply:
One way, though it's a bit hacky, might be to use `--prefix` to move the `node_modules` directory to another place (maybe `.node_modules`), though I don't know whether that can be configured such that `npm` knows about it automatically.

Alternatively, could we do something like `'SB**/*.md'`?